### ISSUE TITLE
hw01

### ISSUE THREAD

**Issue by sherrie9** [STAT545-UBC-students/hw01-CeciliaLe07#3]
Thanks for your reflection, I think the main reason your graph is missing from github documents is that you might have forgotten to knit the file (remember the knit button? ) Only after that the .md can be changed. otherwise whatever graph you produce in .rmd file won't be saved after you close that file.  I dont know why you can't run rmarkdown in r console. I don't think you need to open a new R session to run the codes. Bring the problem to me in person maybe we can figure it out. 

Good readme file with excellent use of various techniques. 

Data exploration is good with good coding style. However, I noticed that you manually edited the .md file to include the plots. Please don't do this in the future. The plots are supposed to be included automatically when you knit the .rmd file. 

Another suggestion is to comment on the output, especially for the plots. Why do you include such plots and what do they tell you?

**Comment by CeciliaLe07** [comment]
Hi @sherrie9 , thank you very much for your feedback. I used the knit button, but even after that I couldn't see my images, so I thougth my assigment was finished and submited my reflection.

However, after that I realized I need to include the directory with all images that R generated when I ran the knit button. So, I only added this directory on my last commit called "submiting files with plots" (before the deadline of this aasigment)  and my images could be showed, but I never typed manually the command to include my graphs on the .md file.

Sorry for the incovenience, I think I should have submit my reflection after I'm sure my last commit was made, but I made a commit after submit my reflection at canvas.

**Comment by sherrie9** [comment]
oo I see. Sorry, I understand what's going on now and should take back what I said about you manually edited .md file.  
Another student has a similar problem today. I think it was due to that she uploads the .md file using the upload bottom to github (without uploading the generated pictures, I suspect) . We fixed the issue when we pushed the .md file to github from local. Are you able to push to github from your local repository? If you do that, you don't have to upload image files yourselves every time when submitting any .md files .

**Comment by CeciliaLe07** [comment]
Yes, from that moment to now I always make almost my commits and all my push from my local repository. I was wondering if the happened situation affected negatively my grade on this assigment, and also if due to the aclaration about what happened is possible to get a greater grade. Thanks a lot!

**Comment by sherrie9** [comment]
Yes you are right, it shouldn't have affected your grade since the issue is cleared up. I will adjust the grade accordingly.
